### PR TITLE
Ensure marmottawrapper has minimal external dependencies

### PR DESCRIPTION
**Pull request by anarchivist**
- Use `#{app_root}/marmotta` as the data directory...for real
- `marmottawrapper` now no longer depends on `curl` and `unzip`
  - `rake marmotta:fetch` uses `open-uri` and grabs a tarball
  - `rake marmotta:install` uses `Archive::Tar::Minitar` and `GZip`
- Update README to describe Marmotta configuration conventions 

Specifically, once Marmotta creates its required files on its first startup, users can then modify the default configuration.

### CODE REVIEW

File: lib/marmottawrapper.rb [hunk 1]
```diff
@@ -27,44 +30,44 @@ def tomcat_dir
...
+      tb = Zlib::GzipReader.new(File.open(tarball, 'rb'))
+      Archive::Tar::Minitar.unpack(tb, app_root.to_path)
     end 
 
     ##
     # Start the Tomcat server with Marmotta 
     def start
-      system "#{tomcat_dir}/bin/startup.sh"
+      system "MARMOTTA_HOME=#{marmotta_dir} #{tomcat_dir}/bin/startup.sh"
+      end
```
Comment (no-reply) [inline lib/marmottawrapper.rb]: ? 

Blame your text editor?

Comment (anarchivist) [inline lib/marmottawrapper.rb]: Good catch; fixed in 3faddf0

**Comment by no-reply** [comment]
:jack_o_lantern:

**Comment by anarchivist** [comment]
:squash: :arrow_right:  :merge:

**Pull request merged**